src/ctia/properties.clj

```diff
@@ -82,6 +78,9 @@
                       "ctia.hook.redis.enabled" s/Bool
                       "ctia.hook.redismq.enabled" s/Bool})
 
+   (st/required-keys {"ctia.access-control.min-tlp" s/Str
+                      "ctia.access-control.default-tlp" s/Str})
```

Comment:
You could use the TLP schema from CTIM

Reply:
Also agreed.  Let's reuse the schema we've got.